internal/output/html.go

```diff
@@ -234,21 +246,24 @@ func processPackageResults(allVulns []HTMLVulnResult, groupIDs map[string]models
 	}
 
 	// Sort packageResults to ensure consistent output
-	sort.Slice(results, func(i, j int) bool {
-		return results[i].Name < results[j].Name
+	slices.SortFunc(results, func(a, b HTMLPackageResult) int {
+		if c := cmp.Compare(a.Name, b.Name); c != 0 {
```

Comment:
a trick you can use here is the cmp.Or() function

Reply:
Good hint, updated!